proto/cosmos/staking/v1beta1/tx.proto

```diff
@@ -62,12 +62,9 @@ message MsgCreateValidator {
   string                   validator_address = 5 [(cosmos_proto.scalar) = "cosmos.AddressString"];
   google.protobuf.Any      pubkey            = 6 [(cosmos_proto.accepts_interface) = "cosmos.crypto.PubKey"];
   cosmos.base.v1beta1.Coin value             = 7 [(gogoproto.nullable) = false];
-  // The orchestrator field is a celes1... string  (i.e. sdk.AccAddress) that
-  // references the key that is being delegated to.
-  string orchestrator = 8;
   // This is a hex encoded 0x EVM public key that will be used by this
   // validator on the target EVM chain.
-  string evm_address = 9;
+  string evm_address = 8;
```

Comment:
I changed the field number to 8 because I don't think we need to think about compatibility at this stage, no one is using the QGB anyway.